tests/test_manipulation.py

```diff
@@ -72,10 +73,13 @@ def test_repeat(arr):
 
 
 def test_result_type():
```

Comment:
currently failing in the internal CI, since no latest dpctl available there, probably we need to update the test by a separate PR

Reply:
For now, I skip the test and put it down to remove skip later.